terraform/modules/enclave/prometheus/variables.tf

```diff
@@ -76,3 +76,7 @@ variable "targets_bucket" {
 variable "prometheus_public_fqdns" {
   type = "list"
 }
+
+variable "logstash_hosts" {
```

Comment:
this variable is called `logstash_hosts` but I'm not sure from reading the code how I would pass more than one host to it.  Can it be either documented (eg with a [variable description](https://www.terraform.io/docs/configuration/variables.html#description-1)) or renamed to `logstash_host` please?